scripts/build.sh

```diff
@@ -23,3 +24,6 @@ set -o pipefail && xcodebuild -project $PROJECT_NAME -scheme "$SCHEME" -destinat
 echo "#### Testing RELEASE configuration for scheme: $SCHEME, with destination: $DESTINATION ####"
 echo "Running: \"xcodebuild -project $PROJECT_NAME -scheme "$SCHEME" -destination "$DESTINATION" -configuration Release ONLY_ACTIVE_ARCH=NO ENABLE_TESTABILITY=YES test | xcpretty -c;\""
 set -o pipefail && xcodebuild -project $PROJECT_NAME -scheme "$SCHEME" -destination "$DESTINATION" -configuration Release ONLY_ACTIVE_ARCH=NO ENABLE_TESTABILITY=YES test | xcpretty -c;
```

Comment:
I think the change to swap this to using xcbeautify got lost in a merge

Reply:
🤦